Frontend/src/pages/AboutPage.js

```diff
@@ -2,13 +2,118 @@ import React from 'react';
+        <br/>
+        <li><strong>Computer Scientists and Algorithm Developers:</strong> Professionals working in computer science, especially those involved in algorithm development, may use databases of arithmetical dynamical systems for applications in computational mathematics, algorithmic analysis, and complexity theory.</li>
+        <br/>
+        <li><strong>Cryptographers:</strong> Researchers in cryptography might be interested in arithmetical dynamical systems for their applications in the design and analysis of cryptographic algorithms. Certain chaotic maps and dynamical systems are used in cryptographic protocols.</li>
+        <br/>
+        <li><strong>Researchers in Control Theory:</strong> Professionals working in control theory, engineering, and systems analysis may explore arithmetical dynamical systems for applications in system control, stability analysis, and related fields.</li>
+        <br/>
+        <li><strong>Mathematical Software Developers:</strong> Developers of mathematical software or tools may reference databases of arithmetical dynamical systems to validate implementations, test algorithms, and ensure the accuracy of computational tools.</li>
+        <br/>
+        <li><strong>Research Institutions and Laboratories:</strong> Academic institutions, research laboratories, and organizations focused on mathematical research may maintain or use databases of arithmetical dynamical systems as part of their research resources.</li>
+    </ul>
+    <h1>Getting started with  DADS:</h1>
+    <p>Visit our <a className = "linkButton" href="https://github.com/oss-slu/dads" target="_blank">Github Page</a>.</p>
+
+    <h2>Dependencies</h2>
+    <p><strong>Python dependencies:</strong> flask, flask_cors</p>
+    <p><strong>Frontend:</strong> Node, React, MUI</p>
+
+    <h2>How to run backend</h2>
+    <p>Run the following command inside the backend folder:</p>
+    <code>python server.py</code>
+
+    <h2>How to run frontend</h2>
+    <p>For the first time, run the following command inside the frontend folder:</p>
+    <code>npm install</code>
+
+    <p>Then, run the following command inside the frontend folder. The website should open up automatically in the browser:</p>
+    <code>npm run start</code>
+
+    <h1>Troubleshooting:</h1>
+    <div>
+    <h2>Problems with Docker:</h2>
+    <p>
+        <a href="https://docs.docker.com/desktop/install/windows-install/" target="_blank">Docker Desktop Installation Guide</a>, 
+        or use the appropriate package manager on a UNIX based system.
+    </p>
+    <p>
+        (Homebrew will likely be the best for Mac, but research your Linux distro to use the package manager it was installed with.
+        The command will likely look like this: <code>sudo PACKAGE_MANAGER_NAME install docker</code>)
+    </p>
+
+    <h3>Setup Docker Container</h3>
```

Comment:
These setup instructions are great, should be easy to replace when we change our database setup